src/common/processUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { Ora } from "ora";
+import * as Sentry from "@sentry/node";
+// @ts-ignore It's necessary for sentry to trace errors. Importing @sentry/tracing patches the global hub for tracing to work.
+import * as Tracing from "@sentry/tracing";
+
+export const isDevelopment = () => process.env.NODE_ENV === "development";
+
+export const setupSentry = () => {
+    if (!isDevelopment()) {
+        Sentry.init({
+            dsn: "https://b4ad6d6db883487ebc914236321bf0e0@sentry.is.canonical.com//50",
+        });
+    }
+} 
+
+export const displayError = (error: Error, spinner: Ora) => {
+    const errorMessage = error.message;
+    errorMessage
+        ? spinner.fail(`${errorMessage}`)
+        : spinner.fail("An error occurred.");
+
+        if(!isDevelopment())  Sentry.captureException(error);
```

Comment:
The only issue I am getting is `displayError` with no error message. Do you think that will be enough for you to fix issues?

![image](https://user-images.githubusercontent.com/2707508/160562690-33d833dd-016a-49a6-af6d-fed842db1eb6.png)


https://sentry.is.canonical.com/canonical/ght/issues/25201/?query=is%3Aunresolved